metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/templates/enrichment-splitjoin.properties.j2

```diff
@@ -1,63 +0,0 @@
-#
```

Comment:
Why are these changes in here?

Reply:
Bah, sorry, just a miss on my part.  reverted